client/src/components/QueryComponent/QueryComponent.scss

```diff
@@ -267,3 +267,10 @@
     }
   }
 }
+
+.ms-TextField--multiline
+.ms-TextField-wrapper{
```

Comment:
Please add a space between the class name and the `{`.

Reply:
Change Ready